### ISSUE TITLE
[ENHANCEMENT]: Show total votes count only

### ISSUE THREAD

**Issue by rajmanna-dev** [vigneshs-dev/Q-Vote#31]
### Description
Show total votes count instead of showing counts for every candidate, its a over transparency issue 😆. We can publish like this after voting ends.

### Screenshots
![Screenshot 2024-10-11 121410](https://github.com/user-attachments/assets/726cad6b-8c1a-4ab0-8964-41e3b6182ddb)

### Additional context

I don't know if this counts shows in a different context. What do you think?

**Comment by vigneshs-dev** [comment]
Yeah . You are right @rajmanna-dev . I placed the counts of each candidate to cast vote for the winning candidate to check whether the users vote count added correctly. As the logic works correctly and the votes are counted perfectly we can remove that part. Instead we can show that in results page as you suggested. Do you want to work on this issue?

**Comment by rajmanna-dev** [comment]
Yeah sure! Assign me on it
